app/src/main/java/ar/scacchipa/twittercloneapp/data/model/TweetsDataWrapper.kt

```diff
@@ -0,0 +1,9 @@
+package ar.scacchipa.twittercloneapp.data.model
+
+import com.google.gson.annotations.SerializedName
+
+data class TweetsDataWrapper(
```

Comment:
please, choose a naming format either has Wrapper or not but to have both styles is confusing

Reply:
The class was moved to the package data.model.response.